NineChronicles.DataProvider/RenderSubscriber.cs

```diff
@@ -846,6 +1038,398 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
                                 var end = DateTimeOffset.Now;
                                 Log.Debug("Stored MigrateMonsterCollection action in block #{index}. Time Taken: {time} ms.", ev.BlockIndex, (end - start).Milliseconds);
                             }
+
+                            if (ev.Action is Grinding grind)
+                            {
+                                _renderCount++;
```

Comment:
What is this variable used for?

Reply:
Tldr, the DP rendersubscriber can't catch up with the chain tip if it inserts data for each action. Therefore, I've set up a rendercount that inserts all accumulated data when the count hits 500(this limit is just randomly selected which seems to be working). In short, the DP inserts data into mysql for every 500 subscribed actions.